external/kafka/src/main/scala/org/apache/spark/streaming/kafka/DirectKafkaInputDStream.scala

```diff
@@ -17,6 +17,8 @@
 
 package org.apache.spark.streaming.kafka
 
+import org.apache.spark.streaming.scheduler.rate.RateEstimator
```

Comment:
Please order imports based on the following convention
https://cwiki.apache.org/confluence/display/SPARK/Spark+Code+Style+Guide